db/migrate/20211019192121_create_cdn_configuration.katello.rb

```diff
@@ -0,0 +1,28 @@
+class CreateCdnConfiguration < ActiveRecord::Migration[6.0]
+  def up
+    create_table :katello_cdn_configurations do |t|
+      t.integer :organization_id
+      t.integer :ssl_ca_credential_id
+      t.text :ssl_cert
+      t.text :ssl_key
+      t.string :username
+      t.string :password
+      t.string :upstream_organization_label
+      t.string :url
+    end
+
+    add_foreign_key :katello_cdn_configurations, :taxonomies, name: 'katello_cdn_configurations_organization_id', column: :organization_id
+    add_foreign_key :katello_cdn_configurations, :katello_content_credentials, name: 'katello_cdn_configurations_ssl_ca_credential_id', column: :ssl_ca_credential_id
+
+    ::Organization.all.each do |org|
+      Katello::CdnConfiguration.where(
+        organization: org,
+        url: org.redhat_provider.repository_url || ::Katello::Resources::CDN::CdnResource.redhat_cdn_url
+      ).first_or_create!
+    end
+  end
```

Comment:
should we drop the redhat provider url column here?

Reply:
Good idea, but I I'd like to file a redmine and do it separately. Reason being is that this code might get ported back to 6.9 perhaps as a hotfix for some customers who want to sync RH content from their 7.0. In order to limit the size+risk `of the change, and especially because the `OrganizationCreator' (thing that replaced the Org::Create dynflow task) isn't in 6.9 and moved creation of providers etc. around, I'd rather have the unused field left over in the code+db than break something. Lemme know if I'm not making any sense :)

Reply:
Taking a look - maybe it's not so bad